src/main/java/seedu/exercise/model/schedule/Schedule.java

```diff
@@ -27,21 +29,31 @@ public Regime getRegime() {
     }
 
     public List<Exercise> getExercises() {
-        return regime.getExercises().asUnmodifiableObservableList();
+        return regime.getRegimeExercises().asUnmodifiableObservableList();
     }
 
     /**
      * Returns true if both {@code schedules} have the same date
      */
-    public boolean isSameSchedule(Schedule schedule) {
-        return date.equals(schedule.date);
+    public boolean isSameResource(Resource otherResource) {
```

Comment:
Include `@Override` for overridden methods

Reply:
done